package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@neynar/nodejs-sdk",
-  "version": "1.21.2",
+  "version": "1.21.3",
```

Comment:
@MrKevinOConnell @Shreyaschorge generally speaking, let's keep the version change isolated from the feature PRs. Helps with a few things
1. It makes it easier to revert and push a new version change on top
2. reduces conflicts in case a PR is stuck in review for longer than expected.